frontend/src/components/ShootDetails/ShootInfrastructureCard.vue

```diff
@@ -219,6 +219,12 @@ export default {
     },
     canLinkToSecret () {
       return this.shootSecretBindingName && this.shootNamespace
+    },
+    customDomainChipText () {
```

Comment:
Why does the name start with custom if the text could be generated. I think `shootDomainType` would be better.